src/modules/minigame/fire_fighters/fire_fighters_mingame.gd

```diff
@@ -155,3 +155,7 @@ func get_fire_at(tile: Vector2i) -> FireFightersMinigameFire:
 
 func is_tile_burning(tile: Vector2i) -> bool:
 	return fires.has(tile)
+
+
+func return_to_overworld() -> void:
+	get_tree().change_scene_to_file("res://modules/settlement/breezekiln/breezekiln.tscn")
```

Comment:
We might want to eventually have some navigation class handle switching between the areas so we don't need to specify exact scene paths in the minigame. The EventBus would simply see that someone is "trying to leave the FireFighters minigame" and listen for that, and then change the scene accordingly.